Pokemon Unity/Assets/start.unity

```diff
@@ -1227,78 +1227,6 @@ GUIText:
     rgba: 4279242768
   m_PixelCorrect: 1
   m_RichText: 1
---- !u!1 &8200580
-GameObject:
```

Comment:
@TeamPopplio Why is there so many deletes on this file?

Reply:
those were there not for it to "function" but for them to be recreated, that's why i renamed them to "Old" (and had them duplicated). Can we leave them there until they're fully/completely replaced/replicated? (they serve a purpose until it can be fully phased out). Unless you're capable of designing the UI without a blueprint? If that's the case, feel free to remove them.

That was my way of "using alpha to make beta better." (Dont have to "abandon things that dont work, just redo it and fix any problems along the way).

Reply:
Added it back, sorry about that.

Reply:
I'd personally just use the old concept image for the pokedex and start from scratch, but that won't be for a bit.